### DISCUSSION TITLE
Deconstruct Order could use improvements.

### DISCUSSION THREAD

**Discussion by AtobaAzul** [FakeFishGames/Barotrauma#13707] [Unstable feedback]
Currently, bots ordered to deconstruct items only get 1 item at a time, regardless if the item is stackable, and if theitem is stackable, it also gets its order cleared when 1 item is removed, leaving the rest of the stack.

This makes it **really** slow and I'd rather bring all the stuff myself.

https://cdn.discordapp.com/attachments/668819422919524352/1224150865888149585/Barotrauma_2024-03-31_21-10-42.mp4?ex=661c7256&is=6609fd56&hm=96b3d364b217ff9151aec82ac694a9214aa1448fc61289e1db73b73256474b53&
![image](https://github.com/FakeFishGames/Barotrauma/assets/94794129/8b80da9d-5cdd-492e-8c62-50fc23aecdaf)

**Comment by zeroinformatique** [comment]
When it works, at all. Sometimes they just stop working. Sometimes they try to use the Outpost's deconstructor and run between both your sub and the outpost, leaving items everywhere.
